web/gds-admin-ui/src/redux/dashboard/actions.js

```diff
@@ -46,9 +46,28 @@ const fetchVaspsApiResponseError = (actionType: string, data: any): Action => ({
 
 const fetchVasps = (): Action => ({
     type: FetchVaspsActionTypes.FETCH_VASPS,
-    payload: { },
+    payload: {},
 });
 
+const fecthRegistrationsReviews = (): Action => ({
+    type: FetchRegistrationsReviewsActionTypes.FETCH_REGISTRATIONS_REVIEWS,
+    payload: {}
+})
+
+const fetchRegistrationsReviewsSuccess = (actionType: string, data: any): Action => ({
+    type: FetchRegistrationsReviewsActionTypes.API_RESPONSE_SUCCESS,
+    payload: {
+        actionType, data
+    }
+})
+
+const fetchRegistrationsReviewsError = (actionType: string, error: any): Action => ({
+    type: FetchRegistrationsReviewsActionTypes.API_RESPONSE_ERROR,
+    payload: {
+        actionType, error
+    }
+})
+
```

Comment:
I noticed this in the previous PR also - is this just to prepare for fetching data from the API?

Reply:
yes I do it like to make things easier with backend communication